README.md

```diff
@@ -1,2 +1,4 @@
 # wg-prometheus
-This repository is used to track progress of WG Prometheus
+This repository is used to track progress of the Prometheus workgroup to design and build full compatibility between OpenTelemetry and Prometheus.
+
+
```

Comment:
```suggestion

This repository is used to track progress of the Prometheus workgroup to design and build full compatibility between OpenTelemetry and Prometheus.
```

in case you ever enable markdownlint here :-)